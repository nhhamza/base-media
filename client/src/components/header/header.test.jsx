import React from 'react';
import { shallow } from 'enzyme';

import Header from './header.component';

describe('Header component (disconnected)', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Header />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
