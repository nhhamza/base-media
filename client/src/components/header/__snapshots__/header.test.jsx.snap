// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component (disconnected) should render correctly 1`] = `ShallowWrapper {}`;
